packages/google_maps_flutter/google_maps_flutter_android/pubspec.yaml

```diff
@@ -2,7 +2,7 @@ name: google_maps_flutter_android
 description: Android implementation of the google_maps_flutter plugin.
 repository: https://github.com/flutter/packages/tree/main/packages/google_maps_flutter/google_maps_flutter_android
 issue_tracker: https://github.com/flutter/flutter/issues?q=is%3Aissue+is%3Aopen+label%3A%22p%3A+maps%22
-version: 2.17.0
+version: 2.17.1
```

Comment:
@stuartmorgan-g should this be a minor version bump not a patch version bump? 
I reviewed https://github.com/flutter/flutter/blob/master/docs/ecosystem/contributing/README.md but didnt see anything more specific than my general understadning of sem-ver.

Reply:
Yes, since it adds new public API semver requires it to be a minor API bump. (That's not a repo policy, it's basic semver.)